frontend/src/scenes/surveys/SurveyEditQuestionRow.tsx

```diff
@@ -229,6 +229,10 @@ export function SurveyEditQuestionGroup({ index, question }: { index: number; qu
                                         const newQuestions = [...survey.questions]
                                         newQuestions[index] = newQuestion
                                         setSurveyValue('questions', newQuestions)
+                                        setSurveyValue(
+                                            'appearance.ratingButtonColor',
+                                            val === 'emoji' ? '#939393' : 'white'
+                                        )
```

Comment:
fixes buggy behavior where we wouldn't update the colors correctly when we toggled the survey values.

Before

![2024-07-03 13 47 40](https://github.com/PostHog/posthog/assets/4853149/751b5289-4fcd-4a0d-b9a0-44f45e5bb235)

After

![2024-07-03 13 45 50](https://github.com/PostHog/posthog/assets/4853149/95edc8f8-70a3-4201-9275-567fac0f3383)